src/turbomind/triton_backend/llama/LlamaTritonModel.cc

```diff
@@ -46,6 +46,17 @@ std::shared_ptr<AbstractTransformerModel> AbstractTransformerModel::createLlamaM
             reader.GetInteger("ft_instance_hyperparameter", "pipeline_para_size"),
             reader.GetInteger("ft_instance_hyperparameter", "enable_custom_all_reduce", 0),
             model_dir);
+    }else if (data_type == "bf16") {
```

Comment:
Miss white space between `}` and `else`
And it's better to put `else if` in a new line for consistency